### PR TITLE
Feature/5 api call for patient detail get

### PR DESCRIPTION
**Pull request by alblopnov**
## Description
The objective of this feature is to implement the get endpoint of the patient details that will be used this week in the development of the application.

## Related Issue(s)
- #5  

## Checklist
- [x] I have read and followed the project's contributing guidelines.
- [x] I have tested the changes locally.
- [x] I have made corresponding changes to the documentation if necessary.
- [x] My code follows the project's code style guidelines.

## Reviewers
- @Thealean28 
- @jmartinacu 

## Screenshots (if applicable)
image

## Notes (if applicable)

### CODE REVIEW

**Review by Thealean28 — DISMISSED** [review]
Good job!

File: tests/api/test_acat_patient.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        "contact_phone": "string",
+        "age": 0,
+        "first_appointment_date": "2024-02-26"
+    }
+
+    Patient.create(session, **patient_data)
+
+    url = f'{settings.API_STR}acat/patient/details/{patient_data["id"]}'
+    response = client.get(url)
+    assert response.status_code == 200
```
Comment (gonzcm) [inline tests/api/test_acat_patient.py:59]: This PR should be reversed and revised.

**Review by gonzcm — CHANGES_REQUESTED** [review]

File: migrations/env.py
Comment (gonzcm) [inline migrations/env.py:1]: This file should not be included.

File: src/core/config.py
Comment (gonzcm) [inline src/core/config.py:1]: This changes should not be included.

File: src/main.py
Comment (gonzcm) [inline src/main.py:1]: This changes should not be included.

**Review by gonzcm — CHANGES_REQUESTED** [review]

File: migrations/env.py
Comment (alblopnov) [reply migrations/env.py:1]: 👍

File: src/core/config.py
Comment (alblopnov) [reply src/core/config.py:1]: 🥇

File: src/main.py
Comment (alblopnov) [reply src/main.py:1]: fixed it already 🐐

**Review by Thealean28 — APPROVED** [review]
Good job! Everything is perfect.

**Review by gonzcm — DISMISSED** [review]
Good job!

**Review by alvarobernal2412 — APPROVED** [review]
This PR is ready to merge now!

**Review by jmartinacu — APPROVED** [review]
Everything look fine

**Pull request merged**